py/server/deephaven/experimental/s3.py

```diff
@@ -26,7 +31,7 @@ class S3Instructions(JObjectWrapper):
     S3Instructions provides specialized instructions for reading from AWS S3.
     """
 
-    j_object_type = _JS3Instructions
+    j_object_type = _JS3Instructions if _JS3Instructions is not None else type(None)
```

Comment:
`_JS3Instructions or type(None)` is more idiomatic.

Reply:
+1